src/mjolnir/landmark_builder.cc

```diff
@@ -232,10 +234,10 @@ std::vector<Landmark> LandmarkDatabase::get_landmarks_by_ids(const std::vector<i
   return landmarks;
 }
 
-std::vector<Landmark> LandmarkDatabase::get_landmarks_by_bbox(const double minLat,
-                                                              const double minLong,
-                                                              const double maxLat,
-                                                              const double maxLong) {
+std::vector<Landmark> LandmarkDatabase::get_landmarks_by_bbox(const double minLong,
+                                                              const double minLat,
+                                                              const double maxLong,
```

Comment:
id use either lon or lng `long` is an integer type so is confusing to be used here